testing/environments/snapshot.yml

```diff
@@ -0,0 +1,40 @@
+# This should test the environment with the latest snapshots
+beat:
+  build: .
+  links:
+    - elasticsearch
+    - logstash
+    - kibana
+  environment:
+    - LS_HOST=logstash
+  container_name: beat
+
+elasticsearch:
+  build: ./docker/elasticsearch
+  dockerfile: Dockerfile-2.2.0
+  command: elasticsearch -Des.network.host=0.0.0.0
+  ports:
+    - "9200:9200"
+  container_name: elasticsearch
+
+logstash:
+  build: ./docker/logstash
+  dockerfile: Dockerfile-2.2.0
+  links:
+    - elasticsearch
+  container_name: logstash
+  environment:
+    - ES_HOST=elasticsearch
```

Comment:
We only need to open it, if we also want to access it from outside the environment. This would be helpful if we want to run for example filebeat on OS X and send it to the environment. Now writing this +1 on this :-) will add it.